src/core/Akka/Actor/Scheduler/TimerScheduler.cs

```diff
@@ -150,6 +151,11 @@ public bool IsTimerActive(object key)
             return _timers.ContainsKey(key);
         }
 
+        /// <summary>
+        /// Retrieves all current active timer keys
+        /// </summary>
+        public ImmutableArray<object> ActiveTimers => _timers.Keys.ToImmutableArray();
```

Comment:
I would just return an `ICollection<object>` or whatever here so we don't have to allocate an immutable array each time.

Reply:
I understand that this is probably done for the purposes of making sure we don't get rugged by the collection changing out from under us, but it's better to take that risk and leave the API un-opionated

Reply:
Done